Test/spec/simd/meta/simd_i8x16_arith.py

```diff
@@ -154,6 +138,19 @@ def full_bin_test_data(self):
             self.range_test_data
         ]
 
+    def get_invalid_cases(self):
+        invalid_tests = super().get_invalid_cases()
+        return self.get_malformed_cases() + invalid_tests
+
+    def get_malformed_cases(self):
+        malformed_cases = [';; i8x16.mul was removed from SIMD spec',
+                           '(assert_malformed (module quote\n'
+                           '  "(func (result v128) '
+                           '(i8x16.mul (v128.const i8x16 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1) '
+                           '(v128.const i8x16 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2)))")\n'
+                           '  "unknown operator")\n\n']
+        return '\n'.join(malformed_cases)
```

Comment:
I don't think we should require `i8x16.mul` to be rejected as malformed: the proposal doesn't prohibit the instruction, it just doesn't include it.

Reply:
OK, test removed.